.coderabbit.yaml

```
@@ -0,0 +1,163 @@
+# Custom review instructions
+instructions: |
+  Focus on:
+  1. Security vulnerabilities in payment processing (RGB Protocol, Lightning Network)
+  2. Performance issues in Three.js game code
+  3. Mobile responsiveness and touch handling
+  4. WebSocket connection stability
+  5. Input validation and sanitization
+  6. Memory leaks in game components
+  7. Bitcoin address validation
+  8. Rate limiting implementation
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)